deploy/operator.yaml

```diff
@@ -28,5 +28,3 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "cert-policy-controller"
-          args:
-            - "--enable-lease=true"
```

Comment:
I know this isn't deployed through the GRC helm chart, but will this deployment detail need to be updated somewhere else? Something with the addon manager? I haven't changed those deployment details before, so I'm not sure where they live

Reply:
You are right. I'll have to remove this line here:
https://github.com/open-cluster-management/klusterlet-addon-operator/blob/56609f79aed359696db77ae4abea67e2449e690c/helm-charts/cert-policy-controller/templates/deployment.yaml#L41